Library/Homebrew/extend/ENV.rb

```diff
@@ -29,7 +29,7 @@ def with_build_environment
 
   def clear_sensitive_environment!
     ENV.each_key do |key|
-      next unless /(cookie|key|token)/i =~ key
+      next unless /(cookie|key|token/password)/i =~ key
```

Comment:
why is that a / not a |

Reply:
@ilovezfs because I suck at typing, lol?